### ISSUE TITLE
TA Feedback: SRS - Pranav

### ISSUE THREAD

**Issue by Inreet-Kaur** [PKALXI/RapidCare#170]
Requirements (Pranav)
The document contains design decisions. For example, it specifically mentions interface elements such as "dictate button" and "a feedback bar".
There seems to be functional requirements for stretch goals. For example, FR13 (...a list of frequently used medicines based on accepted diagnosis...) seems to be related to STG1
Furthermore, there are parts that I cannot understand. Some examples are, "A patient file is requested, the request is denied then a detailed request is submitted" and  "This will be a supplementary auto-complete type of tool again to continue to speed up documentation." and "Auto-completion accuracy is 85+ percent as that the benchmark for
most autocompleting keyboards etc." What is a autocompleting keyboard, and what does the keyboard have to do with suggesting medicine?
I cannot understand several fit criteria, such as "...the data’s latency will depend on the networks’ need..." and "...benchmark for most autocompleting keyboards..."

Traceability matrix (Pranav)
The table in 8 Traceability Matrices and Graphs is not standard and the term "dependency" is not explained.
Some requirements depend on each other: NFR 1 depends on NFR2; NFR2 depends on NFR1.

**Comment by PKALXI** [comment]
There seems to be functional requirements for stretch goals. For example, FR13 (...a list of frequently used medicines based on accepted diagnosis...) seems to be related to STG1

**IN CONFIRMATION**

**Comment by PKALXI** [comment]
PR Linked
